server/src/main/java/com/google/speech/tools/voxetta/servlets/BlobstoreLinkServlet.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+ 
```

Comment:
For files that use prefix-style comments (e.g. # or //) license header comment blocks must not have blank lines in the middle.

Reply:
If I add // to line 2 (and all other blank lines) is this okay? Or should there not be any lines without text?

Reply:
changed to //